server.js

```diff
@@ -1,23 +1,43 @@
-/* eslint global-require: 0 */
+/* eslint comma-dangle: 0, global-require: 0, import/first: 0, no-console: 0 */
 
 const production = (process.env.NODE_ENV === 'production')
+
+import 'babel-polyfill'
```

Comment:
would move this import be first and consolidate the newrelic one to just be `if (process.env.NODE_ENV === 'production') import('newrelic')`